text/src/autogluon/text/automm/__init__.py

```diff
@@ -0,0 +1,9 @@
+__version__ = '0.1'
```

Comment:
Incorrect version, this is not how we specify version.

Instead:

```
try:
    from .version import __version__
except ImportError:
    pass
```

Reply:
Thanks. Revised.